web-app/src/main/webapp/index.html

```diff
@@ -8,12 +8,12 @@
   <link rel="stylesheet" href="css/normalize.css" />
   <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet" />
   <link rel="stylesheet" href="css/style.css" />
+  <script
+  src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDtnOIOa1q7Mwn8OeRkeXdNPxpbaWT2ZFw"
+  type="text/javascript"></script>
```

Comment:
Why did you do this? Just curious!

Reply:
The global variable of infowindow accesses map library, therefore it needs to be loaded before other script file does. To do so, I moved the reference to script upwards and deleted defer keyword

Reply:
not a big deal, but can you somehow wait for the maps library to be loaded before creating your markers/info windows?

Reply:
I think it already does this because I am using synchronous loading for maps api, so subsequent script tags that uses api will not execute until api is fully loaded!